src/modules/aws_vpc/entity/route.ts

```diff
@@ -0,0 +1,75 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { cloudId } from '../../../services/cloud-id';
+import { RouteTable } from './route_table';
+
+@Entity()
+export class Route {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne(() => RouteTable, {
+    onDelete: 'CASCADE',
+    onUpdate: 'CASCADE',
+  })
+  @JoinColumn()
+  routeTable: RouteTable;
+
+  // destinations
+  @Column({ nullable: true })
+  @cloudId
+  DestinationCidrBlock?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  DestinationIpv6CidrBlock?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  DestinationPrefixListId?: string;
+
+  // through
```

Comment:
what does this comment mean, can you clarify?

Reply:
Yes. A Route has two types of values:
- Destination (`DestinationCidrBlock`, `DestinationIpv6CidrBlock`, `DestinationPrefixListId`) which one of the values should be set. They match the destination of a request.
- Through: through what entity should the request be routed? For example it might be a VPC peering connection, an instance id or a NAT gateway.